firebase/database.rules.json

```diff
@@ -0,0 +1,6 @@
+{
+  "rules": {
```

Comment:
We should copy the rules currently in the database. Though it may need a different format